rfc8724.xml

```diff
@@ -25,7 +25,7 @@ b) Applying Static Context Header Compression (SCHC) and Fragmentation for
 Low-Power Wide Area Network (LPWANs) to UDP and IPv6
 
 -->
-    <title abbrev="LPWAN SCHC">Static Context Header Compression (SCHC) and Fragmentation for LPWAN, Application to UDP/IPv6</title>
+    <title abbrev="LPWAN SCHC">SCHC: a framework for header compression and fragmentation for Low-Power Wide Area Networks (LPWANs)</title>
```

Comment:
The intent is to:
- focus on the main point: a generic framework.
- avoid (in the title) the distraction that a simple application to UDP/IPv6 is also described in the same document.
I suggest to use SCHC as a name, not an acronym, in the title and in the rest of the document.
This will avoid all the issues like "SCHC Compression: why is compression mentionned twice?" and "SCHC Fragmentation: is it compression or fragmentation?".

Reply:
Should we add "generic"?  a generic framework

Reply:
The problem of using SCHC as a word is that nobody out of the lpwan group knows what does it means. We need to define the acronym as Carles suggested yesterday and then used it as a word.
One idea could be to add F after SCHC to have the fragmentation inside this acronym: SCHCF pronounces as we do CHIC.

Reply:
Do people need to know what a name means? You know your pressure cooker is a SEB, you don't know it used to mean Société des Emboutisseurs Bourguignons. Your know you (neigbor's) car is a BMW, you don't know it means Bayerische Motor Werke. Why do people need to know what SCHC means? All that we want is them to remember it exists and is useful for compression and fragmentation of IP protocols.